packages/dw-react-controls/src/components/TaxonomyPicker/TaxonomyPickerDialog/TaxonomyPickerDialog.base.tsx

```diff
@@ -75,7 +77,12 @@ export const TaxonomyPickerDialogBase: React.FC<ITaxonomyPickerDialogProps> = (p
 
 	React.useEffect(() => {
 		(async () => {
-			const terms = rfdc()(await provider.getTermTree());
+			const filterOptions: Partial<ITermFilterOptions> = {
+				trimDeprecated: !allowDeprecatedTerms,
```

Comment:
add properties to `useEffect` dependencies
`[provider, allowDeprecatedTerms, allowDisabledTerms]`
(PS: and I think `showRootNode` and `rootNodeKey` should actually be in there as well?)

Reply:
Done 🙂